### PR TITLE
#1825 New script - UTILITIES - Data Access Test

### PR DESCRIPTION
**Pull request by C-Love**
New script to test access to tables or other data structures

### CODE REVIEW

File: utilities/data-access-test.vbs [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+'Closing the connection
+objConnection.Close
+
+Call find_user_name(assigned_worker)
+
+email_subject = "Usage Log Database Access Test Completed for " & assigned_worker
+email_body = "The Usage Log test was successful."
+
+Call create_outlook_email("", "hsph.ews.bluezonescripts@hennepin.us", "", "", email_subject, 1, False, "", "", False, "", email_body, False, "", True)
+MsgBox "Test 1 Successfull"
```
Comment (MDR22) [inline utilities/data-access-test.vbs:110]: Typo - successful

File: utilities/data-access-test.vbs [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+'close the connection and recordset objects to free up resources
+objPENDConnection.Close
+Set objPENDRecordSet=nothing
+Set objPENDConnection=nothing
+
+email_subject = "Cases Pending Database Access Test Completed for " & assigned_worker
+email_body = "The Cases Pending test was successful."
+
+Call create_outlook_email("", "hsph.ews.bluezonescripts@hennepin.us", "", "", email_subject, 1, False, "", "", False, "", email_body, False, "", True)
+MsgBox "Test 2 Successfull"
```
Comment (MDR22) [inline utilities/data-access-test.vbs:151]: Typo - successful

File: utilities/data-access-test.vbs [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+' MAXIS_case_number = "124312"
+'Initial Dialog - Case number
+Dialog1 = ""                                        'Blanking out previous dialog detail
+BeginDialog Dialog1, 0, 0, 190, 85, "Data Test"
+  EditBox 60, 25, 45, 15, MAXIS_case_number
+  ButtonGroup ButtonPressed
+    OkButton 80, 65, 50, 15
+    CancelButton 135, 65, 50, 15
+  Text 5, 10, 185, 10, "Enter a CASE Numberfor the test."
```
Comment (MDR22) [inline utilities/data-access-test.vbs:56]: typo - space between Numberfor

**Review by MDR22 — APPROVED** [review]
Tested a few times and appeared to work as expected. Just a few minor typos.

**Review by IlseFerris — APPROVED** [review]
Looks good, save for what Mark already commented on. I approve thee!

**Comment by C-Love** [comment]
I appreciate the specifics on the correct spelling @MDR22 - it is very helpful 😅

**Pull request merged**